autogpts/autogpt/autogpt/models/utils.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+
+from pydantic import BaseModel
+
+
+class ModelWithSummary(BaseModel, ABC):
+    @abstractmethod
+    def summary(self) -> str:
+        """Should produce a human readable summary of the model content."""
+        pass
```

Comment:
Unimportant; no need for `pass` when there's a docstring + no newline at the end